app/assets/stylesheets/application.css.scss

```diff
@@ -147,21 +151,19 @@ nav {
 
 #post-body {
```

Comment:
Some css conventions here, classes are generally used for styling and ids are mainly for javascript.